src/types/Session.js

```diff
@@ -108,8 +108,19 @@ const Session = `
   type Session_Feedback {
     id: ID!
     content: String!
+    tags: [String!]!
     votes: Int!
 
     createdAt: DateTime!
   }
+
+  input Session_FeedbackTagInput{
+    name: String
+    value: String
+  }
+
+  type Session_FeedbackTag {
```

Comment:
Where is this type used and why does it need name and value? As far as I saw it, you implemented tags as a simple `String` so you should not need these types?